src/app/game-state/achievement.service.ts

```diff
@@ -430,6 +430,24 @@ export class AchievementService {
       },
       unlocked: false
     },
+    {
+      name: "Unlimited Taels",
+      description: "You have unlocked bloodline 4 and now you have unlimited money to spent",
+      hint: "Rumour said that some people are too rich.",
+      check: () => {
+        return this.characterState.bloodlineRank >= 4;
+      },
+      effect: () => {
+        if (this.characterState.bloodlineRank == 4) {
+          this.characterState.bloodlineCost = 10000000000000
+        }
+        
+        if (this.characterState.bloodlineRank == 5) {
```

Comment:
Not sure this will work. I don't think we have anything that guarantees this will trigger when we hit bloodline 5. Probably better to put this logic where the bloodline cost increases in upgradeBloodline.